demos/typescript-app/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Dropdown, DropdownProps } from "kintone-ui-component";
+
+const root = document.getElementById("root") as HTMLElement;
+const dropdownProps: DropdownProps = {
+  items: [
+    {
+      label: "-----",
+      value: "-----",
+    },
+    {
+      label: "Orange",
+      value: "orange",
+    }
+  ],
+  value: "-----",
+  label: "フルーツ一覧",
+  error: "エラーです",
```

Comment:
Please change to English.

label: "Fruit",
error: "Error occurred!",

Please add one more items option as an example.